src/TimeGridEvent.js

```diff
@@ -39,24 +40,36 @@ function TimeGridEvent(props) {
     </div>,
   ]
 
+  const eventStyle = isBackgroundEvent
+    ? {
+        ...userProps.style,
+        top: `${top}%`,
+        height: stringifyPercent(height),
+        width: `calc(${width}% + 10px)`,
+        [rtl ? 'right' : 'left']: `${Math.max(0, xOffset)}%`,
+        opacity: 0.75,
+      }
+    : {
+        ...userProps.style,
+        top: stringifyPercent(top),
+        width: stringifyPercent(width),
+        height: stringifyPercent(height),
+        [rtl ? 'right' : 'left']: stringifyPercent(xOffset),
+      }
+
   return (
     <EventWrapper type="time" {...props}>
       <div
         onClick={onClick}
         onDoubleClick={onDoubleClick}
-        style={{
-          ...userProps.style,
-          top: stringifyPercent(top),
-          [rtl ? 'right' : 'left']: stringifyPercent(xOffset),
-          width: stringifyPercent(width),
-          height: stringifyPercent(height),
-        }}
+        style={eventStyle}
         title={
           tooltip
             ? (typeof label === 'string' ? label + ': ' : '') + tooltip
             : undefined
         }
         className={clsx('rbc-event', className, userProps.className, {
+          'rbc-background-event': isBackgroundEvent,
```

Comment:
hmm maybe it should be `isBackgroundEvent ? 'rbc-background-event' : 'rbc-event'` should give a bit more flexibility for styling, wdt?